components/rmrk/Gallery/Search/SearchBar.vue

```diff
@@ -331,8 +370,19 @@ export default class SearchBar extends mixins(PrefixMixin) {
     const res = this.searched.filter((r) => r.name === value)
     return res.length ? true : false
   }
+
+  private removeSearchHistory(value: string): void {
+    const temp = this.searched.filter((r) => r.name !== value)
+    this.searched = temp
+    localStorage.kodaDotSearchResult = JSON.stringify(this.searched)
+  }
 }
 </script>
+<style>
+a.dropdown-item{
+  padding-right: 1px;
+}
+</style>
```

Comment:
This will be global too, you could try to add a custom class on wrapper `<b-autocomplate>`

Reply:
How about I add something like `.autocomplete .dropdown-item{padding-right: 1px}`?

Reply:
> How about I add something like `.autocomplete .dropdown-item{padding-right: 1px}`? This works when it is not scoped.

Ok, put it in global for futur reference

Reply:
Done